crates/volta-core/src/tool/package/fetch.rs

```diff
@@ -164,12 +163,10 @@ fn npm_pack_command_for(
     session: &mut Session,
     current_dir: &Path,
 ) -> Fallible<ToolCommand> {
-    let args = vec![
-        OsString::from("pack"),
-        OsString::from("--no-update-notifier"),
-        OsString::from(format!("{}@{}", name, version)),
-    ];
-    let mut command = run::npm::command(args, session)?;
+    let mut command = run::npm::command(session)?;
+    command.arg("pack");
+    command.arg("--no-update-notifier");
+    command.arg(format!("{}@{}", name, version));
```

Comment:
Out of curiosity, could you have left it as the previous `Vec` and just done `command.args(args)`? (I think this is equally fine, probably comparable—and equally negligible!—performance-wise, genuinely just curious.)

Reply:
We could have left it as a `vec`, but because of how the previous `args` was typed, there were a bunch of ultimately unnecessary `OsString::from` calls. I originally wanted to switch it to a `Vec<&str>`, but the formatted argument breaks that, so I went with the individual arg approach.

Reply:
Makes sense, that's roughly what I guessed. 👍